edx_lint/pylint/module_trace.py

```diff
@@ -41,6 +41,7 @@ class ModuleTracingChecker(BaseChecker):
 
     def visit_module(self, node):
         """Called for each module being examined."""
+        self    # pylint: disable=pointless-statement
```

Comment:
Why silence rather than removing?

Reply:
@nedbat made this commit

Reply:
I just found https://github.com/openedx/edx-lint/pull/285 -- which I would have seen earlier, but I though 5.2.5 was the latest release. :-)